### PR TITLE
added tests and bump module version

### PR DESCRIPTION
**Pull request by hollesse**

### CODE REVIEW

File: mob_test.go [hunk 1]
```diff
@@ -643,6 +643,37 @@ func TestResetDeleteRemoteWipBranchCommit(t *testing.T) {
...
+	reset(configuration)
+
+	assertOutputContains(t, output, "mob reset --delete-remote-wip-branch")
+	assertMobSessionBranches(t, configuration, "mob/master-green")
+}
+
+func TestResetDeleteRemoteWipBranchCommitBranch(t *testing.T) {
+	_, configuration := setup(t)
+	configuration.WipBranchQualifier = "green"
+	configuration.ResetDeleteRemoteWipBranch = true
```
Comment (gregorriegler) [inline mob_test.go:664]: We could move this down, to the 'act' part, like we did in the other tests.
But I'm fine with this, too.

File: mob_test.go [hunk 2]
```diff
@@ -643,6 +643,37 @@ func TestResetDeleteRemoteWipBranchCommit(t *testing.T) {
 	assertNoMobSessionBranches(t, configuration, "mob-session")
 }
 
+func TestResetCommitBranch(t *testing.T) {
+	output, configuration := setup(t)
+	configuration.WipBranchQualifier = "green"
+
```
Comment (gregorriegler) [inline mob_test.go:649]: I would delete that empty line, but its not a big deal

File: mob_test.go [hunk 3]
```diff
@@ -643,6 +643,37 @@ func TestResetDeleteRemoteWipBranchCommit(t *testing.T) {
...
+	assertOutputContains(t, output, "mob reset --delete-remote-wip-branch")
+	assertMobSessionBranches(t, configuration, "mob/master-green")
+}
+
+func TestResetDeleteRemoteWipBranchCommitBranch(t *testing.T) {
+	_, configuration := setup(t)
+	configuration.WipBranchQualifier = "green"
+	configuration.ResetDeleteRemoteWipBranch = true
+
```
Comment (gregorriegler) [inline mob_test.go:665]: I would delete that empty line, but its not a big deal

**Review by gregorriegler — APPROVED** [review]

File: mob_test.go (hunk 1 above)
Comment (hollesse) [reply mob_test.go:664]: That doesn't work here, as we need this configuration before we do mob start. I can duplicate it to have if in the act part to if you like.

**Pull request merged**